ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -392,27 +392,27 @@
         :read identity
         :format (fn [{change-type :change-type :as route-changes}]
                   (case change-type
-                    :no-traffic
+                    "no-traffic"
```

Comment:
Nitpicking, but I suppose clojure should prefer keyword evaluation before string evaluation:
1. To enable potential performance gains: https://clojure.org/guides/faq#why_keywords
(in addition https://clojuredocs.org/clojure.core/identical_q, https://cljs.github.io/api/cljs.core/#keyword-identicalQMARK)
2. To make catchin mistypings a little (but only litte) bit easier.

As we know backend specql did the conversion, that is not done here so if above is wanted, then result column should go through a `keyword` function before giving it from backend.